config/locales/en.yml

```diff
@@ -432,6 +432,7 @@ en:
     suggestions_feedback: Are these suggestions helpful?
     user_role: Role
     course_details: Course Details
+    course_submitted: Edits of course don't happen until you submit your course
```

Comment:
How about...

"Submitted status is used for education program courses where a course plan must be approved before the project begins. It is only relevant for the ClassroomProgram course type."